packages/@atjson/renderer-commonmark/test/commonmark-test.ts

```diff
@@ -759,6 +760,48 @@ After all the lists
     );
   });
 
+  test("five * are avoided", () => {
+    let document = deserialize(
+      {
+        text: "\uFFFCSpace: the final frontier",
+        blocks: [
+          {
+            id: "B00000000",
+            type: "text",
+            parents: [],
+            selfClosing: false,
+            attributes: {},
+          },
+        ],
+        marks: [
+          {
+            id: "M00000000",
+            type: "bold",
+            range: "[1..6]",
+            attributes: {},
+          },
+          {
+            id: "M00000001",
+            type: "italic",
+            range: "[1..6]",
+            attributes: {},
+          },
+          {
+            id: "M00000002",
+            type: "bold",
+            range: "[6..26]",
+            attributes: {},
+          },
+        ],
```

Comment:
Does this also work if the `marks` list has the italic first?
```
marks: [
          {
            id: "M00000000",
            type: "italic",
            range: "[1..6]",
            attributes: {},
          },
          {
            id: "M00000001",
            type: "bold",
            range: "[1..6]",
            attributes: {},
          },
          {
            id: "M00000002",
            type: "bold",
            range: "[6..26]",
            attributes: {},
          },
        ]
```

Reply:
I ask since CKEditor will produce 
```
<i><strong>Space</strong></i><strong>: the final frontier</strong>
```

Reply:
Marks are sorted in an alphabetic order, so this will always be rotated to be:

Reply:
(Since bold comes before italic alphabetically 🤷)